src/index.js

```diff
@@ -1,17 +1,28 @@
 import Perf from 'performance-node';
+import { flatten } from 'flat';
 
 import pkg from '../package';
 import { addToReport } from './addToReport';
+import { wrap as shimmerHttp } from './shimmerHttp';
+
+const METRIC_PREFIX = '@iopipe/trace';
 
 function getConfig(config = {}) {
-  const { autoMeasure = true } = config;
+  const { autoMeasure = true, autoHttp = {} } = config;
   return {
+    autoHttp: {
+      enabled:
+        typeof autoHttp.enabled === 'boolean'
+          ? autoHttp.enabled
+          : Boolean(process.env.IOPIPE_TRACE_AUTO_HTTP),
```

Comment:
Should probably be `IOPIPE_TRACE_AUTOHTTP_ENABLED` to be consistent with other plugins.

Reply:
Yes

Reply:
@kolanos is `IOPIPE_TRACE_AUTO_HTTP_ENABLED` good for you?

Reply:
Yes, that works.